scylla-macros/Cargo.toml

```diff
@@ -21,3 +21,7 @@ proc-macro2 = "1.0"
 [lints.rust]
 unnameable_types = "warn"
 unreachable_pub = "warn"
+
+[dev-dependencies]
+scylla = { version = "0.15.0", path = "../scylla"}
+scylla-cql = { version = "0.4.0", path = "../scylla-cql"}
```

Comment:
:thinking: This looks like an unexpected place for such dependency. What I'm afraid of is notorious forgetting about bumping versions here. How can we make this more discoverable and easier to remember about?

Reply:
If you don't update those lines when bumping major version of scylla-cql / scylla, then Cargo will yell at you.
If you don't update those lines when bumping minor version, then nothing bad will happen. New minor version will be compatible with the one specified here, so it will just be used.

Reply:
Ok